workflow/snakemake_rules/nextclade.smk

```diff
@@ -102,6 +102,37 @@ rule get_sequences_without_nextclade_annotations:
         echo "[ INFO] Number of {wildcards.reference} sequences to run Nextclade on: $(grep -c '^>' {output.fasta})"
         """
 
+rule download_nextclade_executable:
```

Comment:
Do we still need to do this even though Nextclade is part of the nextstrain/base Docker image? Is this to ensure we always have the latest version of Nextclade in our ncov-ingest runs?

Reply:
Not needed, but also doesn't do harm as far as I see it. We can remove it later. Will create an issue to remove in future.